### PR TITLE
feat: add option for app-with-router

### PR DESCRIPTION
**Pull request by PraveenGandhi**
@3cp ,

feat: app-with-router

continuation of https://github.com/aurelia/new/pull/12

related to https://github.com/aurelia/new/issues/5

### CODE REVIEW

**Comment by PraveenGandhi** [comment]
@3cp , checks are failing, don't know why. Please check.

File: app-with-router/src/my-app.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+<nav>
+  <a goto="welcome">Welcome</a>
+  <a goto="about">About</a>
+</nav>
+
+<au-viewport default="welcome" fallback="missing"></au-viewport>
+
+<div>
```
Comment (3cp) [inline app-with-router/src/my-app.html]: Pls remove this footer.

File: app-with-router/src/my-app.scss__if_sass [hunk 2]
```diff
@@ -0,0 +1,18 @@
+nav {
+	background: #eee;
```
Comment (3cp) [inline app-with-router/src/my-app.scss__if_sass]: Indentation pls. You have `\t` in the file.

**Comment by 3cp** [comment]
Sometimes github actions fails with strange network error. I can rerun it. But you will push soon, so it will check again anyway.

File: app-with-router/src/my-app.html (hunk 1 above)
Comment (PraveenGandhi) [reply app-with-router/src/my-app.html]: done

File: app-with-router/src/my-app.scss__if_sass (hunk 2 above)
Comment (PraveenGandhi) [reply app-with-router/src/my-app.scss__if_sass]: done

**Comment by PraveenGandhi** [comment]
> Sometimes github actions fails with strange network error. I can rerun it. But you will push soon, so it will check again anyway.

all passed now.

**Comment by 3cp** [comment]
Thx. I will do some manual test before merge. After that, I will fill up the missing unit tests and e2e tests setup for this new skeleton.

Will be merged in one or two days.

**Pull request merged**